lapis2-docs/src/components/ConfigGenerator/ConfigGenerator.tsx

```diff
@@ -0,0 +1,32 @@
+import { type Dispatch, type SetStateAction, useState } from 'react';
+import { type Config, ConfigProvider, type PartialConfig } from './configContext.tsx';
+import { ConfigWizard } from './ConfigWizard.tsx';
+import { Results } from './Results.tsx';
+import { UploadConfig } from './UploadConfig.tsx';
+import { load } from 'js-yaml';
+
+export const ConfigGenerator = () => {
+    const [config, setConfig] = useState<PartialConfig | undefined>(undefined);
+
+    if (config === undefined) {
+        return (
+            <div className='not-content text-center '>
+                <button onClick={() => setConfig({ metadata: [] })} className='btn btn-primary'>
+                    New
+                </button>
+                <UploadConfig setConfig={setConfig} />
+            </div>
+        );
+    }
+
+    return (
+        <div className='not-content flex'>
+            <ConfigProvider initialConfig={{ ...config, opennessLevel: 'OPEN' }}>
+                <div className='flex'>
+                    <ConfigWizard />
+                    <Results />
+                </div>
+            </ConfigProvider>
+        </div>
+    );
```

Comment:
Do we want to offer a possibility to go back to "New / Upload" from here other than pressing F5?

Reply:
I've added a ticket. #616